tests/e2e/multicontrolplane/multi_control_plane_suite_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build e2e
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controlplane
+
+import (
+	"testing"
+
+	"github.com/istio-ecosystem/sail-operator/pkg/test/util/supportedversion"
+	k8sclient "github.com/istio-ecosystem/sail-operator/tests/e2e/util/client"
+	"github.com/istio-ecosystem/sail-operator/tests/e2e/util/common"
+	"github.com/istio-ecosystem/sail-operator/tests/e2e/util/env"
+	"github.com/istio-ecosystem/sail-operator/tests/e2e/util/kubectl"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	cl                     client.Client
+	err                    error
+	version                = supportedversion.New
+	namespace              = common.OperatorNamespace
+	deploymentName         = env.Get("DEPLOYMENT_NAME", "sail-operator")
+	controlPlaneNamespace1 = env.Get("CONTROL_PLANE_NS1", "istio-system1")
+	controlPlaneNamespace2 = env.Get("CONTROL_PLANE_NS2", "istio-system2")
+	istioName1             = env.Get("ISTIO_NAME1", "mesh1")
+	istioName2             = env.Get("ISTIO_NAME2", "mesh2")
+	istioCniNamespace      = env.Get("ISTIOCNI_NAMESPACE", "istio-cni")
+	istioCniName           = env.Get("ISTIOCNI_NAME", "default")
+	skipDeploy             = env.GetBool("SKIP_DEPLOY", false)
+	appNamespace1          = env.Get("APP_NAMESPACE1", "app1")
+	appNamespace2a         = env.Get("APP_NAMESPACE2A", "app2a")
+	appNamespace2b         = env.Get("APP_NAMESPACE2B", "app2b")
+	multicluster           = env.GetBool("MULTICLUSTER", false)
+	ipFamily               = env.Get("IP_FAMILY", "ipv4")
+
+	k kubectl.Kubectl
+)
+
+func TestInstall(t *testing.T) {
+	if ipFamily == "dual" || multicluster {
+		t.Skip("Skipping the multi control plane tests")
+	}
+	RegisterFailHandler(Fail)
+	setup()
+	RunSpecs(t, "Multiple Control Planes Test Suite")
+}
+
+func setup() {
+	GinkgoWriter.Println("************ Running Setup ************")
+
+	GinkgoWriter.Println("Initializing k8s client")
+	cl, err = k8sclient.InitK8sClient("")
+	Expect(err).NotTo(HaveOccurred())
+
+	k = kubectl.New("clControlPlane")
```

Comment:
To differentiate between the controlPlane testSuite which is already using  "clControlPlane".

```suggestion
	k = kubectl.New("clMultiControlPlane")
```

Reply:
I was wondering about this. Why is this called "clSomething"?

Reply:
Nevermind. I looked at why this is needed and have now removed it from most of the tests. This is only needed for tests involving multiple clusters.